### ISSUE TITLE
Implement new Figma design to Donate page (mobile view)

### ISSUE THREAD

**Issue by adam-abundis** [hackforla/website#2392]
### Dependency
- #3661

### Overview
we need to implement the Figma design for the mobile version of the Donate Page so that we make the page more clear for the user

<details>
  <summary>Current Donation Page</summary>
  
  <img src="https://user-images.githubusercontent.com/21162229/150029562-4aa5fce7-deef-4053-8ba5-6506b919df4a.png" />
</details>

<details>
  <summary>Donation Page (Figma Design)</summary>
  
  <img src="https://user-images.githubusercontent.com/21162229/150029789-2edca6e0-785f-4899-8e51-a7fb56c669bc.jpg" />
</details>

### Action Items
- [ ] Implement Figma design for the Donate Page (mobile view)
- [ ] Submit a pull request with the proposed changes

### Resources/Instructions
[Figma - Donate Page mobile design (final)](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=11166%3A91022)
[pages/donate.html]( https://github.com/hackforla/website/blob/9a7168d42d43d5de8cc6e3995402bd8746e6b17a/pages/donate.html)
[_sass/components/_donate.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/components/_donate.scss)

**Comment by Zhu-Joseph** [comment]
Progress: Going through Figma and SCSS 
Blockers: None so far
Availability: Random hours here and there
ETA: End of the week

**Comment by Zhu-Joseph** [comment]
2. Blockers: Working on the CSS to adjust styling

**Comment by macho-catt** [comment]
@Zhu-Joseph since this has been inactive for a while, I will unassign you. Please feel free to pick up this issue again when you have the time. Thanks

**Comment by JessicaLucindaCheng** [comment]
@abuna1985 Thanks for working on this issue! Could you please provide a weekly progress report that includes the following:
```
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
```
If you have questions, please let me know. Thank you.

**Comment by JessicaLucindaCheng** [comment]
I left a Slack message on Fri, Feb 4, 2022 for Adam.

**Comment by JessicaLucindaCheng** [comment]
@abuna1985 Since I haven't heard back from you and there has been no progress updates, I'm going to unassign you from this issue and move it back to the Prioritized Backlog.

**Comment by A-BMT02** [comment]
Availability for this week: 6hrs
My estimated ETA for completing this issue:  Next Weekends(Saturday or Sunday)

**Comment by A-BMT02** [comment]
Blocker : How do i get the graphics below ? 
Should i make it using CSS or its provided somewhere ? 

![Screenshot (179)](https://user-images.githubusercontent.com/30394037/161645885-6e1bb21e-f4a2-48a7-81e4-7e228a0d8fad.png)

The design is on figma https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=11166%3A91022

**Comment by JessicaLucindaCheng** [comment]
@A-BMT02 When you have a blocker or question, in addition to asking your question in a comment in your issue, please post in the hfla-site channel on Slack your blocker or question so that we know you have a blocker. We may miss your question/blocker if you just post it in a comment in GitHub.

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty 
On the current website, the graphics for the green donation circles is an svg image. Did you want the green circles to still be an image or did you want it changed to be using scss and code?

**Comment by SAUMILDHANKAR** [comment]
> @ExperimentsInHonesty On the current website, the graphics for the green donation circles is an svg image. Did you want the green circles to still be an image or did you want it changed to be using scss and code?

@JessicaLucindaCheng Bonnie said to ask design for an image of it at Sunday meeting.

**Comment by JessicaLucindaCheng** [comment]
@phuonguvan This is the issue related to the green circles for the mobile design on the Donate page we discussed today on (Sun, Apr 10). Please let me know when you fix the problem in the Figma so we can download the circles as an image with all the text.

**Comment by phuonguvan** [comment]
@JessicaLucindaCheng I couldn't understand the problem with the green circles on the Figma. The preview's image looked perfectly fine to me unless you wanted it to be converted into a different format? When I downloaded the preview, the image has all of the texts unless you want to change it to change the dark layout when it is exported. Please tell me what is wrong with the issue!

**Comment by phuonguvan** [comment]
@JessicaLucindaCheng Is the image's preview correct? 
![Frame 1 (2)](https://user-images.githubusercontent.com/100561599/163466890-ebdde378-85e7-4627-b54d-d00ac7329ed7.png)
https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17065%3A104582

**Comment by JessicaLucindaCheng** [comment]
@phuonguvan Yes, that looks good. Thank you so much for your help.

**Comment by JessicaLucindaCheng** [comment]
> Blocker : How do i get the graphics below ? Should i make it using CSS or its provided somewhere ?
> 
> ![Screenshot (179)](https://user-images.githubusercontent.com/30394037/161645885-6e1bb21e-f4a2-48a7-81e4-7e228a0d8fad.png)
> 
> The design is on figma https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=11166%3A91022

You can download the image from here: https://drive.google.com/drive/folders/1Pd07M-1hocCBrISiQ3x136-FivC9NLh1

**Comment by JessicaLucindaCheng** [comment]
@A-BMT02 
> Blocker : How do i get the graphics below ? Should i make it using CSS or its provided somewhere ?
> 
> ![Screenshot (179)](https://user-images.githubusercontent.com/30394037/161645885-6e1bb21e-f4a2-48a7-81e4-7e228a0d8fad.png)
> 
> The design is on figma https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=11166%3A91022

You can download the image from here: https://drive.google.com/drive/folders/1Pd07M-1hocCBrISiQ3x136-FivC9NLh1

**Comment by A-BMT02** [comment]
Thank you @JessicaLucindaCheng 
Working on it

**Comment by SAUMILDHANKAR** [comment]
@A-BMT02 Please share your progress update. Also, let us know if you are planning to attend any of the team meetings next week so that our team meeting requirements can be satisfied which apply to all the members in the website team. Thank you.
Note: Will leave a slack message and email as well.

**Comment by GRISONRF** [comment]
Availability: 10/04/22 all day.
ETA: 10/06

**Comment by blulady** [comment]
Hey,
I am moving this to the in progress column. Please be sure to move the issues you assign yourself into the In Progress column on the Project Board in the future.
Thank You,
Sarah

**Comment by GRISONRF** [comment]
I haven't made any progress with this issue. I will unassign and move it back to the 'Prioritize Backlog' so it can be re-assigned to someone else. Something came up and I won't have time to work on this for the next few days.

**Comment by jyaymie** [comment]
Availability: 3 hours per day
ETA: 10/21/22 EOD

**Comment by jyaymie** [comment]
Implementation of the new Figma design involves increasing the font size of the text in the green circles. This part of the Donate page, however, is an image. (See line 34 of [website/donate.html](https://github.com/hackforla/website/blob/9a7168d42d43d5de8cc6e3995402bd8746e6b17a/pages/donate.html).)

I'd like to know if I am to re-create the image using HTML and SCSS or if there is a new image available for use.

I am working in the `implement-mobile-design-for-donate-page-2392` branch on my forked repo.

Any assistance is appreciated. Thank you!

**Comment by JessicaLucindaCheng** [comment]
@jyaymie You can download the image from here: https://drive.google.com/drive/folders/1Pd07M-1hocCBrISiQ3x136-FivC9NLh1

It's the donation-chart-for-mobile.svg

**Comment by jyaymie** [comment]
> @jyaymie You can download the image from here: https://drive.google.com/drive/folders/1Pd07M-1hocCBrISiQ3x136-FivC9NLh1
> 
> It's the donation-chart-for-mobile.svg

Ah, there it is! Thanks so much, @JessicaLucindaCheng!

**Comment by jyaymie** [comment]
Text alignment in the mockup is different from how it is on the current website. Here is an example:

<details><summary>Mockup of the Final Version</summary>
<img width="403" alt="Screen Shot 2022-10-18 at 2 43 45 PM" src="https://user-images.githubusercontent.com/104411072/196550870-76b905da-4bc2-4ae0-a1e3-98f76001a253.png">
</details> 

<details><summary>Current Donation Page</summary>
<img width="386" alt="Screen Shot 2022-10-18 at 2 44 24 PM" src="https://user-images.githubusercontent.com/104411072/196550930-d1a7720c-d31a-4d7d-ab4f-dbf1b2e0039c.png">
</details> 

Should I implement only what is explicitly noted on Figma or should I match all aspects of the mockup as closely as possible?

**Comment by jyaymie** [comment]
_**UPDATE**_
1. Progress: I re-ordered content in the donation section and updated the donation chart image.
2. Blockers: Those changes I made are actually written out on Figma, but I am wondering if I need to also implement other changes that are not specifically described. These include text alignment, a box shadow on the gif, and spacing between text and images. Following last night's Developers meeting, I will wait for more direction at Sunday's All Team meeting.
3. Availability: Flexible all days except Saturday
4. ETA: EOD 10/26/22

**Comment by jyaymie** [comment]
During today's All Team meeting, it was revealed that more changes needed to be made to the design. This issue should be picked back up when the redesign is complete.

Changes to the current design include:
- Replacing the top "Donate to Hack for LA" section with the "Running Hack for LA costs money" section
- Correcting the font size of the section titles
- Creating a way to clearly define the different sections

**Comment by ExperimentsInHonesty** [comment]
- We originally made one include and used it in two places in this issue
https://github.com/hackforla/website/issues/2451

- Now we are going to change things up a bit on the new redesign.  Please add this to a meeting agenda for product and dev leads to discuss https://github.com/hackforla/website/issues/3661

**Comment by blulady** [comment]
> * We originally made one include and used it in two places in this issue
>       [Update donate section on multiple pages to be the same component #2451](https://github.com/hackforla/website/issues/2451)
> 
>     * Now we are going to change things up a bit on the new redesign.  Please add this to a meeting agenda for product and dev leads to discuss [Redesign  Donate page (mobile view)  #3661](https://github.com/hackforla/website/issues/3661)

@ExperimentsInHonesty The meeting we are having in January? Or was this resolved in last Monday's meeting?

**Comment by blulady** [comment]
I believe this issue can be closed since #3983 has been created.

**Comment by ExperimentsInHonesty** [comment]
- This issue has been replaced by https://github.com/hackforla/website/issues/3983
